pkg/cmd/container/list.go

```diff
@@ -112,7 +130,7 @@ func (x *ListItem) Label(s string) string {
 	return x.LabelsMap[s]
 }
 
-func prepareContainers(ctx context.Context, client *containerd.Client, containers []containerd.Container, options types.ContainerListOptions) ([]ListItem, error) {
+func prepareContainers(ctx context.Context, client *containerd.Client, containers []containerd.Container, cMap map[string]string, options types.ContainerListOptions) ([]ListItem, error) {
```

Comment:
`cMap` needs a comment to explain its keys and values.

Reply:
done

Reply:
Doesn't seem done